lib/omninav/builder.rb

```diff
@@ -211,28 +211,6 @@ def build_search
     </select>
 
     <div id="cu_search_box">
-      <form action="%<domain>ssearch-results/index.aspx">
```

Comment:
This whole div gets wiped out in the javascript so I don't know why this stuff was in here in the first place.